docs/en_us/platform_api/source/shared/courses_response_values.rst

```diff
@@ -0,0 +1,223 @@
+.. _Course:
+
+Course
+*******
+
+* count (integer): The number of courses in the catalog.
+* next (string): The URL for the next page of results.
+* previous (string): The URL for the previous page of results.
+* results (array): See :ref:`Results`.
+
+.. _Results:
+
+Results
+*******
+
+Parent: ``course``
```

Comment:
I don't know if "parent" is the correct term; I was trying to indicate the hierarchical structure.